cpp/src/arrow/util/bpacking_default.h

```diff
@@ -31,6 +31,7 @@ namespace internal {
 
 inline const uint32_t* unpack1_32(const uint32_t* in, uint32_t* out) {
   uint32_t inl = util::SafeLoad(in);
+  inl = arrow::BitUtil::ToLittleEndian(inl);
```

Comment:
Shouldn't this be `FromLittleEndian`? (it will produce the same code, but it seems logically more accurate)

Reply:
Ah, you are right! I will update it soon.

Reply:
Done